### PR TITLE
Add supported Country check

### PR DESCRIPTION
**Pull request by iekadou**
Add isCountrySupported function

### CODE REVIEW

File: src/TIN.php [hunk 1]
```diff
@@ -109,6 +109,16 @@ public static function fromSlug(string $slug): TIN
         return new self($slug);
     }
 
+    public static function isCountrySupported(string $countryCode): bool
+    {
+        foreach (self::$algorithms as $algorithm) {
+            if (true === $algorithm::supports($countryCode)) {
```
Comment (drupol) [inline src/TIN.php:115]: Shouldn't we uppercase the `$countryCode` by default?

Comment (iekadou) [reply src/TIN.php:115]: I thought about that as well, but the supports function automatically does that for us:
`
    final public static function supports(string $country): bool
    {
        return strtoupper($country) === strtoupper(static::COUNTRYCODE);
    }
`

Comment (drupol) [reply src/TIN.php:115]: Oh fair enough.

**Review by drupol — CHANGES_REQUESTED** [review]
Do you mind fixing the code style `./vendor/bin/grumphp run --tasks=phpcsfixer` ?

**Comment by drupol** [comment]
Thank you!

**Pull request merged**